### DISCUSSION TITLE
8/3 안건

### DISCUSSION THREAD

**Discussion by gitul0515** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#76] [01.Agenda]
* 하나의 PR에 두 개 이상의 issue 내용을 넣어도 되는지?

**Comment by Hyevvy** [comment]
- 이슈번호로 브랜치를 땄으니까 pr도 이슈당 하나가 좋다고 생각합니다. 
특히 현재 스쿼시머지라서(여러 커밋이 대표 커밋 하나로 들어가니까요) 이슈번호대로 브랜치따는게 좋아보여요!

-  수정이면 fix니까 더더욱 커밋메시지가 다르기도 하구요 !!
